virt-v2v/cold/entrypoint

```diff
@@ -87,22 +84,11 @@ if [ "$V2V_source" == "vCenter" ] ; then
           -io "vddk-thumbprint=$V2V_thumbprint"
       )
   fi
+  args+=(-- "$V2V_vmName")
```

Comment:
don't we need to append above (line 81)?

Reply:
I mean, for consistency...

Reply:
no, because line 81 is only when VDDK is available.

Reply:
> no, because line 81 is only when VDDK is available.

let me rephrase, wouldn't it be better to replace line 81 with:
```golang
args+=(
```